docs/use-cases/deploy-ml.md

```diff
@@ -3,6 +3,63 @@ title: "Train and deploy image classification models"
 linkTitle: "Train and deploy classification models"
 weight: 50
 type: "docs"
-layout: "empty"
-canonical: "ml/"
+tags: ["data management", "data", "services"]
+no_list: true
+description: "Use Viam's machine learning capabilities to train image classification models and deploy these models to your machines."
+image: "/ml/training.png"
+imageAlt: "Machine Learning"
+images: ["/ml/training.png"]
 ---
+
+Create and deploy an image classification model onto your machine with Viam's machine learning (ML) capabilities.
+Manage the classification model fully on one platform: collect data, create a dataset and label it, train the model, and deploy it.
+Then, test if your model works for classifying objects in a camera stream or existing images with the `mlmodel` classification model of vision service.
```

Comment:
@npentrel Still might sound a little awkward but I tried to make more classification-centric